app/utils/permissions/types.ts

```diff
@@ -0,0 +1,22 @@
+import type { OrganizationRoles } from "@prisma/client";
+
+export enum PermissionAction {
+  create = "create",
+  read = "read",
+  update = "update",
+  delete = "delete",
+  checkout = "checkout",
+  checking = "checkin",
+}
+export enum PermissionEntity {
```

Comment:
I think we should add the rest as well:
- tags
- categories
- locations
- custom fields
- workspaces
- team members

I think thats it. Idk if I am missing something

Reply:
yes, its really a basic draft, just to show the idea, i have to add all these.